src/backend/marsha/bbb/api.py

```diff
@@ -64,7 +64,9 @@ def get_permissions(self):
                 & (core_permissions.IsTokenInstructor | core_permissions.IsTokenAdmin)
             ]
         elif self.action in ["retrieve"]:
-            permission_classes = [ResourceIsAuthenticated]
+            permission_classes = [
+                ResourceIsAuthenticated | IsClassroomOrganizationAdmin
```

Comment:
Do we need also something like `IsClassroomConsumerSiteAdmin` and `IsClassroomPlaylistAdmin` ?

Reply:
Maybe… wdyt @lunika ?

Reply:
Consumer site no but Playlist yes.